app/src/org/commcare/heartbeat/HeartbeatRequester.java

```diff
@@ -156,7 +156,9 @@ private static void parseUpdateToPrompt(JSONObject latestVersionInfo, UpdateToPr
                     if (latestVersionInfo.has("force")) {
                         forceString = latestVersionInfo.getString("force");
                     }
-                    HiddenPreferences.setLatestCommcareVersion(versionValue);
+                    if (updateType == UpdateToPrompt.Type.APK_UPDATE) {
+                        HiddenPreferences.setLatestCommcareVersion(versionValue);
```

Comment:
Feels like the terminology here is really bad. 
`setLatestCommcareVersion` name sounds like it's for `commcare` playstore version. And `setLatestAppVersion` sounds like it's for `ccz app` version. But in reality they are complete opposites.

Reply:
how are they opposite ?

Reply:
Well, here we're setting the latest ccz version into `setLatestCommcareVersion`. And we'll set  the latest commcare version into `setLatestAppVersion`. 
But in our onboarding docs, we read that `commcare` means the engine and `commcare app` means the ccz. And we mostly call the `ccz` as `app` and commcare as `commcare`. Right?

Reply:
> here we're setting the latest ccz version into setLatestCommcareVersion

we are setting the apk version here

Reply:
Ahh damn! Sorry for the confusion!